upstream-community-operators/pmem-csi-operator/0.8.0/pmem-csi-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,271 @@
+      - name: pmem-csi-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: pmem-csi-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                app: pmem-csi-operator
+                name: pmem-csi-operator
+                pmem-csi.intel.com/webhook: ignore
+            spec:
+              containers:
+              - command:
+                - /usr/local/bin/pmem-csi-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: pmem-csi-operator
+                - name: GODEBUG
+                  value: x509ignoreCN=0
+                image: intel/pmem-csi-driver:v0.8.0
+                imagePullPolicy: IfNotPresent
+                name: pmem-csi-operator
+                resources: {}
+                securityContext:
+                  readOnlyRootFilesystem: true
+                volumeMounts:
+                - mountPath: /tmp
+                  name: tmp
+              serviceAccountName: pmem-csi-operator
+              volumes:
+              - emptyDir: {}
+                name: tmp
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          - services
+          - services/finalizers
+          - serviceaccounts
+          - endpoints
+          - events
+          - secrets
+          - pods
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - daemonsets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - roles
+          - rolebindings
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          verbs:
+          - get
+        serviceAccountName: pmem-csi-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - PMEM-CSI
+  - PMEM-CSI operator
+  - Persistent Memory
+  - PMEM
+  links:
+  - name: PMEM-CSI
+    url: https://intel.github.io/pmem-csi/0.8
+  maintainers:
+  - email: olev.kartau@intel.com
+    name: Olev Kartau
+  - email: patrick.ohly@intel.com
+    name: Patrick Ohly
+  - email: amarnath.valluri@intel.com
+    name: Amarnath Valluri
+  maturity: alpha
+  provider:
+    name: Intel® Corporation
+    replaces: pmem-csi-operator.v0.7.0
+    version: 0.8.0
```

Comment:
@avalluri version and replaces should be indented to one level to the left

Reply:
@mvalahtv Thank you so much. I fixed it now.